src/ui/default-prop-types.js

```diff
@@ -8,13 +8,12 @@ export const translationProps = {
 };
 
 export const sectionDisplayProps = {
-  docKey: PropTypes.string.isRequired,
-  sectionKey: PropTypes.string.isRequired,
```

Comment:
Small clarification here:
- `sectionKey` was never used in either display or edit section cases
- `docKey` was only used in some of the edit section cases, I changed it into the generic `containerParentId` (aka document key or lesson._id)

Reply:
When we overhaul the file picker component, this can disappear completely (it's currently used to set the initial folder to show)

Reply:
Yes, I noticed that it's only used by the components that need to upload to S3 🤔